jdkhttp/README.md

```diff
@@ -72,6 +72,25 @@ then create a yaml from the UI using the content
 of kubernetes/deployment_registry.yaml
 
 changing the image with the name of your docker image.  
+
+#### Build and using a local registry 
+Create the image
+```sh
+oc new-build --binary --strategy=docker --name openshift-kie-springboot
+oc start-build openshift-kie-springboot --from-dir=. --follow
+```
+
+Get the image
+```sh
+oc get is/openshift-kie-springboot -o template --template='{{range .status.tags}}{{range .items}}{{.dockerImageReference}}{{end}}{{end}}'
```

Comment:
Please describe meanings of these placeholders.

Reply:
Reverted